statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
This isn't right, will break on any dev version. Actually my version was wrong too, but at least it won't crash. Maybe use ``re.match(r'\d[.]\d+[.]\d+', pversion)`` plus check if there are any characters left afterwards (then it's an rc/dev/alpha/beta).